srcpkgs/OpenJK/template

```diff
@@ -0,0 +1,47 @@
+# Template file for 'OpenJK'
+pkgname=OpenJK
+version=0.0.20210922
+revision=1
+archs="i686 x86_64"
```

Comment:
Just remove this line and let it fail in CI for now.
Musl should be easy to fix (I did so at least for myself 3 years ago, can do it again)
Also OpenJK runs fine on non x86 systems, I tested it some time ago on aarch64.

Reply:
good to know, thank you